libraries/lib-music-information-retrieval/tests/MirTestUtils.h

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+#include "MirAudioReader.h"
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <fstream>
+#include <numeric>
+#include <string>
+#include <vector>
+
+namespace MIR
+{
+
+// Some tests, such as benchmarking and visualization, are not meant to be run
+// on CI. This variable is used to disable them.
+static constexpr auto runLocally = false;
```

Comment:
So I understand you change this to true, run locally, and then decide just two magic number thresholds to put into the application, in `MusicInformationRetrieval.h`.  The very last thing in `GetMeterUsingTatumQuantizationFit.cpp` is use of those thresholds.

Maybe that header file should have a comment about how the numbers are experimentally derived.